### PR TITLE
Add diagram.

### PR DESCRIPTION
**Pull request by stbof**
DOC-4889

### CODE REVIEW

File: v22.1/refresh.md [hunk 1]
```diff
@@ -15,20 +15,19 @@ CockroachDB does not support materialized views that are refreshed on [transacti
...
 
-## Syntax
+## Synopsis
 
-~~~
-REFRESH MATERIALIZED VIEW [CONCURRENTLY] view_name [WITH [NO] DATA]
-~~~
+<div>
+{% remote_include https://raw.githubusercontent.com/cockroachdb/generated-diagrams/release-{{ page.version.version | replace: "v", "" }}/grammar_svg/refresh_materialized_views.html %}
+</div>
```
Comment (taroface) [inline v22.1/refresh.md:22]: Is there a reason this needs div tags?

File: v22.1/refresh.md [hunk 2]
```diff
@@ -15,20 +15,19 @@ CockroachDB does not support materialized views that are refreshed on [transacti
...
 
 ## Parameters
 
  Parameter | Description
 -----------|-------------
-`CONCURRENTLY` | (*Default behavior*) This keyword is a no-op, added for PostgreSQL compatibility. All materialized views are refreshed concurrently with other jobs.
+`opt_concurrently` | `CONCURRENTLY` (*Default behavior*) This keyword is a no-op, added for PostgreSQL compatibility. All materialized views are refreshed concurrently with other jobs.
```
Comment (taroface) [inline v22.1/refresh.md:28]: I know that "no-op" is a commonly used phrase, but do you think it bears spelling out the meaning for a general reader?

**Review by taroface — APPROVED** [review]
LGTM, with 2 questions.

File: v22.1/refresh.md (hunk 2 above)
Comment (taroface) [reply v22.1/refresh.md:28]: Also, should we not italicize "Default behavior" per our style usage?

File: v22.1/refresh.md (hunk 1 above)
Comment (stbof) [reply v22.1/refresh.md:22]: I just copied it from another page.

File: v22.1/refresh.md [hunk 3]
```diff
@@ -15,20 +15,19 @@ CockroachDB does not support materialized views that are refreshed on [transacti
...
 `view_name` | The name of the materialized view to refresh.
-`WITH NO DATA` | Drop the query results of the materialized view from storage.
-`WITH DATA` | (*Default behavior*) Refresh the stored query results.
+`opt_clear_data` | `WITH DATA` (*Default behavior*) Refresh the stored query results. <br>`WITH NO DATA` Drop the query results of the materialized view from storage.
```
Comment (stbof) [inline v22.1/refresh.md:30]:
```suggestion
`opt_clear_data` | `WITH DATA` (Default behavior) Refresh the stored query results. <br>`WITH NO DATA` Drop the query results of the materialized view from storage.
```

File: v22.1/refresh.md (hunk 2 above)
Comment (stbof) [inline v22.1/refresh.md:28]:
```suggestion
`opt_concurrently` | `CONCURRENTLY` (Default behavior) This keyword has no effect. It is present for PostgreSQL compatibility. All materialized views are refreshed concurrently with other jobs.
```

**Pull request merged**